pandapower/create.py

```diff
@@ -139,7 +139,8 @@ def create_empty_network(name="", f_hz=50., sn_mva=1, add_stdtypes=True):
                    ("type", dtype(object)),
                    ("closed", "bool"),
                    ("name", dtype(object)),
-                   ("z_ohm", "f8")],
+                   ("z_ohm", "f8"),
+                   ("in_ka", "f8")],
```

Comment:
I think its a little different from lines, since it is not meant to be the maximum thermal current, but rather the current where the switch trips (in case of a fuse or a circuit breaker). I checked wikipedia where it is called In: https://en.wikipedia.org/wiki/Fuse_(electrical)#Rated_current_IN. Of course switches aren't always fuses, but it seemed the most fitting parameter name.

Maybe we need another parameter like max_i_ka for the maximum thermal current that a switch can safely carry/interrupt?

Reply:
thank you for clarification! I think it is important to include this information in the documentation, so that to know which current is meant.